src/libraries/Microsoft.PowerFx.Core/Types/Enums/EnumStore.cs

```diff
@@ -184,7 +192,7 @@ public static void RegisterTuple(Tuple<string, string, string> tuple, Dictionary
             }
         }
 
-        public static bool TryGetLocalizedEnumValue(string enumName, string enumValue, out string locValue)
+        public bool TryGetLocalizedEnumValue(string enumName, string enumValue, out string locValue)
```

Comment:
Thinking a bunch of these should probably be `internal` now that we're exposing this. Subclasses should be able to extend the dictionary but not modify the lookup functions, imo

Reply:
Marked the functions as internal instead of public.